src/components/ProjectPreview.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useTheme } from 'styled-components'
+import { PreviewText, PreviewWrapper, PreviewTitle } from './styles/ProjectPreview.styled'
+
+type Props = {
+  title: string
+  text: string
+  hasAnimation?: boolean
+  onClick?: React.MouseEventHandler<HTMLDivElement>
+}
+
+const defaultProps = {
+  hasAnimation: false,
+  onClick: () => undefined,
```

Comment:
undefined is sufficient

Reply:
92c245dd8781d90e0d89f8cd2bad5f03c6ecf6e0